### ISSUE TITLE
Inaccessible headings on basic config page

### ISSUE THREAD

**Issue by paulGeoghegan** [getsentry/sentry-docs#9857]
### Steps to Reproduce

1. open https://docs.sentry.io/platforms/javascript/guides/sveltekit/configuration/options/
2. Turn on NVDA or another screen reader,
3. Try to read the individual configuration options.

### Expected Result

dsn etc

### Actual Result

common options permalink

### Additional Info

Pretty much the <ConfigKey name=""> elements are not accessible the part inside the name attribute should be exposed to screen readers but it is not.
